### ISSUE TITLE
Enhancement for keywords

### ISSUE THREAD

**Issue by commander-trashdin** [jonspalding/lisp-paredit#36]
Changing 

       (loop :for i :from 0 :below n
          :when (oddp i)
          :do (format t "~a is odd, " i)
          (funcall do-something-else i))```

to

     (loop :for i :from 0 :below n
           :when (oddp i)
           :do (format t "~a is odd, " i)
               (funcall do-something-else i))

**Comment by observant2** [comment]
Also this:

```lisp
(defmethod check-collisions ()
  (run-system (((pos 'c-position)
                (col 'c-collision)))
              (let ((shapes (c-collision-collision-shapes col)))
                (dolist (s shapes)
                        (cond ((typep s 'circle) (bla))
                          ((typep s 'box) (blub)))))))
```

The `let` form could be indented less, but especially the `cond` cases should be aligned properly.

**Comment by observant2** [comment]
Okay, now looking into it, I found a solution to the problem I'm facing.
I removed `cond` from the `indentation forms` in paredit's settings. Also I added `dolist` there. My code looks much better now. Maybe you should have a look at that list, too.

**Comment by neil-lindquist** [comment]
I added a note to the README about the Indentation forms.

**Comment by neil-lindquist** [comment]
For more advanced indentation work, I opened #38.  Swank provides a bunch of information on how to indent correctly (or close there to), but it needs to be setup in a way that can be accessed by the indenter.

I'm going to close this unless there are specific issues that can be fixed seperately to issue #38 .
